packages/dashboards/spec/kpi-error.visual.ts

```diff
@@ -0,0 +1,52 @@
+import { Atom, Camera } from "@nova-ui/bits/sdk/atoms";
+import { Helpers } from "@nova-ui/bits/sdk/atoms/helpers";
+import { browser, by, element } from "protractor";
+import { TestPage } from "./test.po";
+import { ConfiguratorAtom } from "./configurator/configurator.atom";
+
+const name: string = "Kpi Error";
+
+describe(`Visual tests: Dashboards - ${name}`, () => {
+    let camera: Camera;
+    let configurator: ConfiguratorAtom;
+    const page = new TestPage();
+
+    beforeAll(async () => {
+        await Helpers.prepareBrowser("test/kpi/error");
+        configurator = Atom.findIn(ConfiguratorAtom, element(by.className(ConfiguratorAtom.CSS_CLASS)));
+
+        camera = new Camera().loadFilm(browser, name);
+    });
+
+    it(`${name} - Default look`, async () => {
+        await camera.turn.on();
+
+        await camera.say.cheese(`${name} - Default`);
+
+        await camera.turn.off();
+    }, 100000);
+
+    it(`${name} - Configurator error message types`, async () => {
```

Comment:
We might want to ask Dima about structuring a visual test suite with multiple `it` blocks. Last I heard, we were supposed to have just a single `it` block for some reason, but maybe that has changed since we moved to Percy.